src/codeu/chat/server/View.java

```diff
@@ -46,15 +46,15 @@ public View(Model model) {
 
   @Override
   public Collection<User> getUsers(Collection<Uuid> ids) {
-    return model.userById(intersect(ids, false), "ASC");
+    return model.userById(ids, false, true);
```

Comment:
Do you have any calls to this method where the ordering is false? If not, then just remove the sorting parameter for this and other methods like it. The main idea is to only include code you need right now.